### PR TITLE
Add "a" & Remove "instead!" from "Not a company/applicant? Register a…

### PR DESCRIPTION
**Pull request by Korny94**
…s an/a *** instead!"

Add the word "a" before "company"
Remove the word "instead!" after "company" and "applicant"

At:

/pages/auth/register/company/index.html
/pages/auth/register/applicant/index.html
/pages/auth/register/index.html

### CODE REVIEW

File: .vscode/settings.json
Comment (Jean-Christoffer) [inline .vscode/settings.json:1]: Why is settings.json being changed ? the issue doesnt seem to be related to settings.json

File: package-lock.json
Comment (Jean-Christoffer) [inline package-lock.json:1]: I dont think package-lock.json was part of the issue you were trying to solve?

Comment (Korny94) [reply package-lock.json:1]: I only edited the html files inside /pages/auth/register. The other files must have been changed automatically when I was installing the project:

## Installing and starting the project

Installing
```
npm i
```

Build
```
npm run build
```

Run locally
```
npm run dev
```

File: .vscode/settings.json
Comment (Korny94) [reply .vscode/settings.json:1]: I only edited the html files inside /pages/auth/register. The other files must have been changed automatically when I was installing the project:

## Installing and starting the project

Installing
```
npm i
```

Build
```
npm run build
```

Run locally
```
npm run dev
```

**Review by Jean-Christoffer — CHANGES_REQUESTED** [review]
Almost there : ) 
if i understand the issue correctly, only a "a" and remove the string "instead" should be changed/added. 
/pages/auth/register/company/index.html
/pages/auth/register/applicant/index.html
/pages/auth/register/index.html

but 5 files are being changed in the pr, please take a look : ) 
you can use git status to see what files you have changed then git add "filename" to add the file/files you want to commit and merge

File: package-lock.json
Comment (Jean-Christoffer) [reply package-lock.json:1]: ah okey, will take a better look when i get home

**Review by Jean-Christoffer — COMMENTED** [review]
I unstaged the unwanted files ".vscode/settings.json" and "package-lock.json".

Now its back to only the three html files you were working on : )

**Review by Jean-Christoffer — APPROVED** [review]
Fixed

**Pull request merged**